lib/ts/recipe/session/types.ts

```diff
@@ -134,7 +129,10 @@ export type TypeNormalisedInput = {
     errorHandlers: NormalisedErrorHandlers;
     antiCsrf: "VIA_TOKEN" | "VIA_CUSTOM_HEADER" | "NONE";
 
-    getTokenTransferMethod: (input: { req: BaseRequest; userContext: any }) => "cookie" | "header";
+    getTokenTransferMethod: (input: {
+        req: BaseRequest;
+        userContext: any;
+    }) => "cookie" | "header" | "MISSING_AUTH_HEADER";
```

Comment:
should we make it all caps or all smallcase? Having smallcase and caps is strange - but you can keep it this if you want.